### PR TITLE
tests: Add docker run stdin integration test.

### LINKED ISSUE #74: Add docker run with stdin
Add a docker integration test where stdin can be handled

### PR DESCRIPTION
**Pull request by GabyCT**
This will test stdin with docker run.

Fixes #74

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: integration/docker/run_test.go [hunk 1]
```diff
@@ -291,10 +295,21 @@ var _ = Describe("run", func() {
...
+			stdout, stderr, exitCode = dockerRun(args...)
 			Expect(exitCode).To(Equal(1))
 			Expect(stdout).To(BeEmpty())
 			Expect(stderr).To(ContainSubstring("ls: /etc/foo: No such file or directory"))
 		})
 	})
+
+	Context("stdin using run", func() {
+		It("should not display the stdin", func() {
+			Skip("Issue https://github.com/clearcontainers/runtime/issues/932")
```
Comment (chavafg) [inline integration/docker/run_test.go]: @GabyCT I see that this issue is not reproducible on kata, so we can remove the skip.

**Comment by GabyCT** [comment]
@chavafg changes applied thanks

**Comment by chavafg** [comment]
lgtm

File: integration/docker/run_test.go [hunk 2]
```diff
@@ -291,10 +295,20 @@ var _ = Describe("run", func() {
...
-			stdout, stderr, exitCode := dockerRun(args...)
+			stdout, stderr, exitCode = dockerRun(args...)
 			Expect(exitCode).To(Equal(1))
 			Expect(stdout).To(BeEmpty())
 			Expect(stderr).To(ContainSubstring("ls: /etc/foo: No such file or directory"))
 		})
 	})
+
+	Context("stdin using run", func() {
+		It("should not display the stdin", func() {
```
Comment (jcvenegas) [inline integration/docker/run_test.go]: The test name should be, "should display to stderr?"

**Comment by GabyCT** [comment]
@jcvenegas changes applied

**Comment by jcvenegas** [comment]
lgtm

**Pull request merged**